src/commcare_cloud/ansible/roles/kafka/templates/server.properties.j2

```diff
@@ -117,3 +123,6 @@ zookeeper.connect={% for host in groups['zookeeper'] %}{{ host }}:{{ zookeeper_c
 
 # Timeout in ms for connecting to zookeeper
 zookeeper.connection.timeout.ms=1000000
+
+inter.broker.protocol.version={{kafka_inter_broker_protocol_version}}
+log.message.format.version={{kafka_log_message_format_version}}
```

Comment:
it will start at `0.8.2.2` until you update the params in public.yml. If you look at the upgrade docs you'll see that the params are only changed after the upgarde.

Reply:
I see. thanks